packages/pill/stories/Button.stories.tsx

```diff
@@ -1,54 +1,81 @@
-import React from 'react';
-import { Pill } from '../src';
-import { IconSearch16, IconPlus16 } from '@warp-ds/icons/react';
+import React from 'react'
+import { Pill } from '../src'
+import { IconSearch16, IconPlus16 } from '@warp-ds/icons/react'
 
-const metadata = { title: 'Buttons/Pill' };
-export default metadata;
+const metadata = { title: 'Buttons/Pill' }
+export default metadata
 
 export const Default = () => {
-  return <Pill label="Example" />;
-};
+  return (
+    <Pill
+      aria-label='a pill that shows default behaviour'
+      label='Example'
+    />
+  )
+}
 
 export const DefaultCanClose = () => {
-  return <Pill label="Example" canClose onClose={() => alert('remove')} />;
-};
+  return (
+    <Pill
+      aria-label='a pill that you can close and that shows default behaviour'
+      label='Example'
+      canClose
+      onClose={() => alert('remove')}
+    />
+  )
+}
 
 export const Suggestion = () => {
-  return <Pill label="Example" suggestion />;
-};
+  return (
+    <Pill
+      aria-label='a pill with suggestion'
+      label='Example'
+      suggestion
+    />
+  )
+}
 
 export const SuggestionCanClose = () => {
   return (
-    <Pill label="Example" suggestion canClose onClose={() => alert('remove')} />
-  );
-};
+    <Pill
+      aria-label='a pill with suggestion and that you can close'
+      label='Example'
+      suggestion
+      canClose
+      onClose={() => alert('remove')}
+    />
+  )
+}
 
 export const PillOnClickAndOnClose = () => {
   return (
     <Pill
-      label="Example"
+      aria-label='a pill that is clickable and that you can close'
+      label='Example'
       onClick={() => alert('pill is clicked')}
       suggestion
       canClose
       onClose={() => alert('close is clicked')}
     />
-  );
-};
+  )
+}
 
 export const PillsWithIcon = () => {
   return (
-    <div className="flex space-x-8">
+    <div className='flex space-x-8'>
       <Pill
+        aria-label='a pill with a search icon'
         icon={<IconSearch16 />}
         onClick={() => alert('onClick event')}
```

Comment:
In the [ticket](https://nmp-jira.atlassian.net/browse/WARP-332) it was mentioned that we should use role="img" and aria-label instead of `title`, which I _think_ was regarding the use of Pill with icon in storybook. I don't think we have the possibility to hide `title` element from the svg at this point so we can just have role and aria-label on the Pill.